src/glog/logging.h.in

```diff
@@ -1195,15 +1195,15 @@ GLOG_MSVC_POP_WARNING()
     LogStream(const LogStream&);
     LogStream& operator=(const LogStream&);
     base_logging::LogStreamBuf streambuf_;
-    int ctr_;  // Counter hack (for the LOG_EVERY_X() macro)
+    size_t ctr_;  // Counter hack (for the LOG_EVERY_X() macro)
```

Comment:
Why `size_t` instead of `uint64`? The former depends on the target platform. `uint64` is more consistent with the changes to parameter types.